### DISCUSSION TITLE
Incorrect classification of used language in github repo. shows 80% HTML instead of Typescript

### DISCUSSION THREAD

**Discussion by mischulz14** [github-linguist/linguist#6126] [Classification] [answered]
![18_10_22_10_22_24](https://user-images.githubusercontent.com/96014669/196376773-1cac2341-d247-49b0-8ae6-bd203a02e779.jpg)
![18_10_22_10_20_48](https://user-images.githubusercontent.com/96014669/196376436-c36647ff-2ce7-468f-880a-0835a19df322.jpg)
![18_10_22_10_21_20](https://user-images.githubusercontent.com/96014669/196376509-d0f5fde6-bc53-4fa3-b64f-a3343c20bf33.jpg)

## Describe the bug
One of my github repositories ( https://github.com/mischulz14/nextjs-ecommerce-store) shows that 80% of my project is written in HTML, even though almost the whole project is written in TS.
If I press onto HTML on the used lanuages I get this message: "We couldn’t find any code matching '' in [mischulz14/nextjs-ecommerce-store](https://github.com/mischulz14/nextjs-ecommerce-store)"

## What I tried 
I tried configuring a .gitattributes file, where html should be ignored. This is working within the repository, but on my profile readme this change didn't work, because my most used language in total is still HTML with almost 60%.

**Comment by mischulz14** [comment, marked as answer]
For any future reference: I solved it. I had a huge playwright HTML file which didn't show up in the repo. I deleted the file and this fixed everything

**Comment by hoghweed** [comment]
I have the same issue unfortunately. if I click on the HTML language detected to filter the files, I cannot see any file in HTML. I don't know how to solve it, any suggestion without relying on .gitattributes?

**Reply by lildude** [reply]
@hoghweed this really should be a new discussion as this one has been resolved. The most likely explanation is your repo contains files that are written in a language that falls into the HTML group, eg Ruby '.erb` files. If you can't work out what is happening, please start a new description linking to the affect repo @hoghweed and we can take a look.
